.github/workflows/pythonpackage_future3.yml

```diff
@@ -6,13 +6,11 @@ on:
         - 'future3/**'
     paths:
         - '**.py'
-        - '**.py.*'
   pull_request:
     branches:
         - 'future3/**'
```

Comment:
You didn't change it, but i think this filter is obsolete, as this class will only exist on future3 branches.
Should we remove it?

Reply:
I would keep it until we make future3 the default